### DISCUSSION TITLE
Integrate warning in the `task` status

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#305] [Feedback & Feature Proposal]
It might be interesting for some users, regarding some situations to add a warning in the `task` status. This warning will not prevent the task from succeeding, but could inform the users.

Ex:
- when cutting the document because too many words are present in the field (65 635 in v0.24.0 and later): https://github.com/meilisearch/MeiliSearch/issues/1770#issuecomment-965632815

**Comment by gmourier** [comment]
Thanks, @curquiza, this is interesting feedback! I think we could go further and define other information of this type that could be useful. The `details` field of a task object could be well suited to this type of information as well. 

We could also put them in stdout with a dedicated format.
